### DISCUSSION TITLE
Microsoft Edge version?

### DISCUSSION THREAD

**Discussion by linheyang** [AutomaApp/automa#90] [Q&A]
I was wondering if a Microsoft Edge version was planned. Thanks!

**Comment by Kholid060** [comment]
If you're using the chromium version of the Microsoft Edge, you can install the extension by opening the [chrome webstore page](https://chrome.google.com/webstore/detail/automa/infppggnoaenmfagbfknfkancpbljcca/)

**Reply by Dayu0815** [reply]
Excuse me!
Can "Automa" be used as a shortcut in the iPhone?

**Reply by Kholid060** [reply]
What do you mean by "shortcut in the iPhone"?

**Reply by Dayu0815** [reply]
Is it possible to operate like this, is it a shortcut function that can be placed on the iphone?
https://www.youtube.com/watch?v=awp_QTCNJqM
